appserver/connectors/connectors-runtime/src/main/java/com/sun/enterprise/resource/allocator/LocalTxConnectorAllocator.java

```diff
@@ -81,17 +80,6 @@ public ResourceHandle createResource()
 
             return resource;
         } catch (ResourceException ex) {
-            Object[] params = new Object[]{spec.getPoolInfo(), ex.toString()};
-            _logger.log(Level.WARNING, "poolmgr.create_resource_error", params);
```

Comment:
Is the deletion of this log as intended? (delegate log to the caller who catches the PoolingException?

Reply:
Yes, because the exception is wrapped and thrown again several times. In effect when something broke, I had 5 times the same error in the server.log file, yet once with a broken message.

But based on your comment it seems to me that some paths may result in hiding the exception, so for sure I will revisit all paths and ensure that the exception is logged exactly once including stacktrace.

Reply:
TODO for next commits of this PR: catch and process the exception, don't repeat logging and rethrowing.

Reply:
I have revisited that one more. It is finally called just from one place in ConnectionPool, which throws the exception including the cause OR sleeps and gives it another try. Now, before the sleep, it should log it as a warning including some message what will happen next.